pages/blog/build-a-mega-menu-with-react-bootstrap.mdx

```diff
@@ -0,0 +1,292 @@
+---
+title: "Build a Navbar Mega Menu Using React Bootstrap"
+date: "2021-05-13"
+author: "Teal Larson"
+pullQuote: "Syncing ain't easy."
+image: "build-a-mega-menu-with-react-bootstrap/210512-mega-menu.png"
+tags: [engineering]
+---
+
+Having clear and accessible navigation is huge for website conversions. Sites with poor navigation are frustrating to use. Nested navigation menus are a common way to help keep top-level navigation to a minimum, but they can have major usability issues. A better way to handle a large number of links in a dropdown is to create a [mega menu](https://www.nngroup.com/articles/mega-menus-work-well/). Recently, we gave our site navigation a face lift using mega menus.
+
+Since both our [core product](https://github.com/grouparoo/grouparoo) and our [website](https://github.com/grouparoo/www.grouparoo.com) are open source here at Grouparoo, let’s dive in and take a look at how we built out a mega menu using our stack.
+
+## First, What’s a Mega Menu?
+
+This!
+
+<Image
+  centered
+  alt="Completed Navbar at grouparoo.com"
+  src="build-a-mega-menu-with-react-bootstrap/final-on-www.png"
+  width={845}
+  height={277}
+/>
+
+A mega menu is a way to have multiple columns (or more!) of items within your dropdown menu. Some key things to consider:
+
+1. Make sure you have enough items within your dropdown to warrant a mega menu
+2. Figure out how you want to arrange the items within your menu
+
+Recently, we’ve been adding a lot of content to our site. We wanted to link more pages in our navigation, but still keep the top level navigation bar minimal. So we went with a mega menu.
+
+## Let’s Build
+
+For this tutorial, we'll be building this copycat mega menu:
+
+<Image
+  centered
+  alt="Copycat Navbar we'll be making"
+  src="build-a-mega-menu-with-react-bootstrap/finished-tutorial.png"
+  width={845}
+  height={269}
+/>
+
+Coffee themed, because of course. I've created a [basic boilerplate](https://github.com/grouparoo/nextjs-react-bootstrap-boilerplate)
```

Comment:
If you want to highlight the boilerplate repo, you could use a <Button> or <Alert> (yay mdx!)